plop-templates/component_js.hbs

```diff
@@ -1,20 +1,14 @@
-import { getPlayById } from 'meta/play-meta-util';
-
 import PlayHeader from 'common/playlists/PlayHeader';
 import './{{camelCase name}}.{{style}}';
 
 function {{pascalCase name}}(props) {
-  // Do not remove the below lines. 
-  // The following code is to fetch the current play from the URL
-  const { id } = props;
-  const play = getPlayById(id);
 
   // Your Code Start below.
 
```

Comment:
Lets create a sample js and ts play to check nothing is breaking!

Reply:
we have all the play coming from backend so i created one but not able to test it. i would likely to get more info about how to test it.